spec/controllers/archive_faqs_controller_spec.rb

```diff
@@ -39,10 +38,20 @@
     context "when logged in as a regular user" do
       before { fake_login_known_user(user) }
 
-      it "redirects to the user preferred locale when the locale param is invalid" do
+      it "redirects to the default locale when the locale param is invalid" do
```

Comment:
I think it might make understanding this easier if it's wrapped in a context as well. Something like
```rb
context "when the set locale preference feature flag is off" do
```